src/App/Components/Common/Triangle.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Proptypes from 'prop-types';
+
+export class Triangle extends Component {
```

Comment:
this one should be a functional component

Reply:
Why ?